src/plugin.ts

```diff
@@ -85,12 +85,12 @@ export const newSamplePlugin = () => {
     spec,
     { noConnection },
   ) => {
-    pluginClient.spec = parseSpec(spec);
```

Comment:
the `package` command and some others pass an empty spec and `noConnection: true` so we should skip parsing the spec for `noConnection: true`